bash/scripts/locate_release_candidate.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+main() {
+  envPropsFilepath="${BATS_TEST_DIRNAME}/tmp/env.properties"
+  if [ -n "${JENKINS_HOME}" ]; then
+    envPropsFilepath="${WORKSPACE}/env.properties"
+  fi
+
+  tag=''
+  if [ -n "${TAG}" ]; then
+    echo "TAG set to '${TAG}', attempting release of this tag..."
+    tag="${TAG}"
+  else
```

Comment:
cuold we change this logic to use an `elif` instead to lower the cyclomatic complexity?